osvr/RenderKit/RenderManagerD3D11ATW.h

```diff
@@ -489,6 +517,21 @@ namespace osvr {
                 std::vector<osvr::renderkit::RenderBuffer> renderBuffers;
                 size_t numRenderInfos = renderInfo.size();
 
+                if (renderInfo.size() == 0) {
+                    m_log->error()
+                        << "RenderManagerD3D11ATW::"
+                        << "RegisterRenderBuffersInternal: renderInfo was zero length.";
+                    return false;
+                }
+
+                // @todo: we need to get the actual device for the given render info associated
+                // with the given buffer being registered. Since we support re-using the buffers
+                // for more than one eye, then we need to actually track this somehow, but we
+                // can't with the given parameters without just making assumptions
+                // for now, just use the first eye's device, but this may not work with
+                // multi-display HMDs
+                auto atwDevice = renderInfo[0].library.D3D11->device;;
```

Comment:
Substantive; how about if we do i % numRenderInfos as was done in the previous implementation, rather than always assuming 0?  That should work in both single-HMD and multi-HMD?  Or maybe we track how many buffers have been locked and increment each time we get a new one instead of using i?  Not thrilled about breaking multi-HMD compatibility.  If we can maintain this, go ahead and merge.